### PR TITLE
improvement: ZENKO-2576 optimize crrExistingObject

### PR DESCRIPTION
**Pull request by naren-rajendran**
Improvements to `crrExistingObjects` script.

- Removed unnecessary serialization and deserialization.
- Removed extra mongo query in `metadaUtil.getMetadata`

### CODE REVIEW

File: crrExistingObjects.js [hunk 1]
```diff
@@ -104,7 +104,7 @@ function _markObjectPending(bucket, key, versionId, storageClass,
         }, log, next),
         (mdRes, next) => {
             objMD = new ObjectMD(mdRes);
-            const mdBlob = objMD.getSerialized();
+            const mdBlob = objMD.getValue();
```
Comment (jonathan-gramain) [inline crrExistingObjects.js]: The name `mdBlob` is confusing because it's not a blob anymore but a JS object, I suggest changing the variable name.

**Review by jonathan-gramain — APPROVED** [review]
Generally speaking, it can be useful to say a few words about the kind of improvement in the PR, or what benefit we can expect from the improvements proposed.

**Review by alexanderchan-scality — APPROVED** [review]

File: crrExistingObjects.js (hunk 1 above)
Comment (naren-rajendran) [reply crrExistingObjects.js]: Updated comment with improvements made and changed the variable name.

**Review by ilkescality — APPROVED** [review]

**Review by anurag4DSB — APPROVED** [review]

**Review by rahulreddy — APPROVED** [review]

**Comment by naren-rajendran** [comment]
@bert-e approve

**Pull request merged**